sdks/python/apache_beam/dataframe/frame_base.py

```diff
@@ -318,8 +318,8 @@ def wrapper(self, *args, **kwargs):
 
 def maybe_inplace(func):
   @functools.wraps(func)
-  def wrapper(self, inplace=False, **kwargs):
-    result = func(self, **kwargs)
+  def wrapper(self, *args, inplace=False, **kwargs):
```

Comment:
That could happen only with methods where the first kwarg is `inplace` right? I wouldn't think there's many of those but I suppose there could be.

So what's the right thing to do here? We could use inspect to find `inplace` in the args/kwargs of `func` and handle it correctly. Or more simply just never use this in methods where we need to pass through args.

Reply:
We shouldn't use this in places where we may need to pass args. (Actually, if we're just passing a`*args`, we can't generally detect `inplace` safely at all).

Reply:
If we passed the base type as in populate_defaults we could inspect it's method. Seems like a lot of trouble.. but I'm just worried that there's some edge case where we're dropping non-keyword args, and it would be great to catch that possibility at import time.

Reply:
I removed this change and instead handle `inplace` in the new methods, PTAL